gaffer-as-a-service/gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/controller/GraphController.java

```diff
@@ -68,14 +68,9 @@ public class GraphController {
     private GetNamespacesService getNamespacesService;
     @Autowired
     private GetGaaSGraphConfigsService getStoreTypesService;
-//    @Autowired(required = false)
-//    private AuthService authService;
```

Comment:
Has this been deleted?

Reply:
Yes. AuthService no longer exists inside of rest.